src/transformers/models/idefics2/modeling_idefics2.py

```diff
@@ -603,6 +604,26 @@ class Idefics2PreTrainedModel(PreTrainedModel):
     _supports_sdpa = True
     _supports_cache_class = True
 
+    def __init__(self, *args, **kwargs):
```

Comment:
Throws informative exception if the user tries to load a model with `attn_implementation="sdpa`